src/builder/commands/composer.rs

```diff
@@ -316,7 +317,7 @@ fn find_conf_dirs() -> Result<Vec<PathBuf>, scan_dir::Error> {
     ScanDir::dirs().skip_symlinks(true).read("/vagga/root/etc", |iter| {
         iter.filter(|&(_, ref name)| name.starts_with("php"))
         .flat_map(|(ref entry, _)| {
-            ScanDir::dirs().read(entry.path(), |iter| {
+            ScanDir::dirs().walk(entry.path(), |iter| {
```

Comment:
Ubuntu messed up with the config directory in xenial, now it's `/etc/php/7.0`. Without the recursive walk the config directory would not be found